phalcon/acl/adapter/memory.zep

```diff
@@ -612,20 +613,16 @@ class Memory extends Adapter
 
 	/**
 	 * Return an array with every role registered in the list
-	 *
-	 * @return Phalcon\Acl\Role[]
 	 */
-	public function getRoles()
+	public function getRoles() -> <Role[]>
 	{
 		return this->_roles;
 	}
 
 	/**
 	 * Return an array with every resource registered in the list
-	 *
-	 * @return Phalcon\Acl\Resource[]
 	 */
-	public function getResources()
+	public function getResources() -> <$Resource[]>
```

Comment:
`Resource` is a keyword in Zephir (as is `Inline`). See https://github.com/phalcon/cphalcon/blob/2.0.x/phalcon/assets/resource.zep#L31 and https://github.com/phalcon/cphalcon/blob/2.0.x/phalcon/assets/inline.zep#L31.